package.json

```diff
@@ -4,13 +4,22 @@
   "version": "0.3.0",
   "description": "A small badge-like piece of UI with a customizable content and style.",
   "main": "spine-badge.html",
+  "repository": "https://github.com/SpineElements/spine-badge.git",
+  "license": "Apache-2.0",
+  "scripts": {
+    "build:styles": "spine-sass-transpiler spine-badge-css.scss",
+    "test": "polymer test --module-resolution=node --npm"
+  },
   "dependencies": {
-    "@polymer/polymer": "^3.0.0",
-    "@polymer/paper-styles": "^3.0.0-pre.18"
+    "@material/elevation": "^0.35.0",
+    "@polymer/lit-element": "^0.5.1"
   },
   "devDependencies": {
-    "wct-browser-legacy": "^0.0.1-pre.11",
+    "@polymer/iron-component-page": "^3.0.0-pre.19",
+    "@polymer/iron-demo-helpers": "^3.0.0-pre.18",
     "@webcomponents/webcomponentsjs": "^2.0.0",
-    "@polymer/iron-demo-helpers": "^3.0.0-pre.18"
+    "spine-sass-transpiler": "github:SpineElements/spine-sass-transpiler#implementation",
```

Comment:
Let's also consider documenting (on the cookbook) the main findings/instructions/guides on Lit element development, such as:
- Quite obviously, but we might need to reiterate that we extend LitElement (not PolymerElement), describe some other key basics, and/or possibly some references with the very basic Lit element creation guide(s).
- Styling:
   - How do we compile scss, convention on where to place the converted files?
   - Is it always necessary to have styles in scss (we can consider letting omitting them for simplicity in some places, which ones)?
   - How is interoperation with the standard Polymer/MCW/MCWC themes made (see the related comments in scss file below). We can at least document what we know so far and then update as we gain more knowledge/experience.
- How do we write tests (e.g. the same Polymer way or not), any references to the respective guides?
   - How do we run the tests (Polymer CLI/WCT)?
- Any general info/references for a new component writer needed to write the simplest basis for a component with all parts (dependencies/dev dependencies, js implementation with styles, any specific configurations relative to Polymer 3.
   - How is it best to document -- some references, maybe a boilerplate to start with, explicit documentation notes?

Let's discuss.